docs/src/modules/components/MaterialFreeTemplatesCollection.js

```diff
@@ -17,56 +18,68 @@ import { pascalCase } from 'docs/src/modules/utils/helpers';
 import sourceMaterialTemplates from 'docs/src/modules/material/sourceMaterialTemplates';
 import codeSandbox from 'docs/src/modules/sandbox/CodeSandbox';
 import stackBlitz from 'docs/src/modules/sandbox/StackBlitz';
+import { sxChip } from 'docs/src/modules/components/AppNavDrawerItem';
 
 const sourcePrefix = `${process.env.SOURCE_CODE_REPO}/tree/v${process.env.LIB_VERSION}`;
 
-function layouts(translatation) {
+function layouts(translation) {
   return [
     {
-      title: translatation('dashboardTitle'),
-      description: translatation('dashboardDescr'),
+      title: translation('dashboardTitle'),
+      description: translation('dashboardDescr'),
       href: '/material-ui/getting-started/templates/dashboard/',
       source: `${sourcePrefix}/docs/data/material/getting-started/templates/dashboard`,
       hasDarkMode: true,
     },
     {
-      title: translatation('marketingPageTitle'),
-      description: translatation('marketingPageDescr'),
+      title: translation('dashboardToolpadTitle'),
```

Comment:
Interleaving different types of content feels like a no-no.

Reply:
Do you mean we should create a separate section for Toolpad Core templates on this page, or create a separate page for them, or do you mean that the Toolpad-specific copy should be stored separately from the rest of the content (not in `translations.json`?)

Reply:
I agree that putting Toolpad template in it's own section will help users distinguish between the templates. We were earlier thinking that it would be better if the user could see Toolpad template before scroll, but due to the design revamp of this page, scrolling is inevitable. So, a separate section will help visually differentiate.


We'll replace the dashboardLayout demo with the hosted Toolpad core template. 👍

Reply:
Yeah, but this should be in Toolpad's docs, not in Material UI's docs. It's not a Material UI feature.

Reply:
Listing Toolpad template here will only make it more discoverable to the Material UI users who are looking for templates. Some of them may be interested in it, so I think we should have it here, at the bottom, in a section of its own. 
Also, the templates docs page attracts many users, so not listing it here, would be a missed opportunity.

Reply:
> Listing Toolpad template here will only make it more discoverable to the Material UI users who are looking for templates. Some of them may be interested in it, so I think we should have it here, at the bottom, in a section of its own.

Referencing Toolpad here makes sense to me, a person reaching for a template has about a 25% probability to be a user we build Toolpad for (internal tool 0.5 * simple use cases 0.5). It's also at the right time.

However,

- for the majority of the people (75%) it's noise
- for the people that use the different product it creates confusion as to what is what
- for the team maintaining Toolpad, it makes them update places outside of their scope

So It seems that the only viable option is to reference Toolapd like we do for the template market.